hack/deploy-kubevirt.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 set -exo pipefail
 
+if [ "$GITHUB_ACTIONS" == "true" ]; then
+    for node in $(./cluster/kubectl.sh get node --no-headers  -o custom-columns=":metadata.name"); do
+        docker exec -t $node bash -c "echo 'fs.inotify.max_user_watches=1048576' >> /etc/sysctl.conf"
+        docker exec -t $node bash -c "echo 'fs.inotify.max_user_instances=512' >> /etc/sysctl.conf"
+        docker exec -i $node bash -c "sysctl -p /etc/sysctl.conf"
```

Comment:
can we please stop writing code that only works on docker ? 

does this happen elsewhere in this project ?

Reply:
it is WIP, please wait for the review

Reply:
ok seems the problem is that the first one out of CNAO / kubevirt is deployed fine, but the 2nd one (tried both orders) isnt
i can add logs to make sure it is memory, but it looks like that

we might want to check if OVN use self-hosted runners
and we can consider moving to prow imo at least as first step